### DISCUSSION TITLE
Roughly Enough Items 8.3 Update

### DISCUSSION THREAD

**Discussion by shedaniel** [shedaniel/RoughlyEnoughItems#958] [Announcements]
This is Roughly Enough Items 8.3 Update, this update introduces the Tag Category, Quick Craft Tooltips, Collapsable Entries API, Config Addons API, and more...

YouTube Video: https://youtu.be/XKMITh9otVs

- https://github.com/shedaniel/RoughlyEnoughItems/issues/512
![2022-06-28_02 34 34](https://user-images.githubusercontent.com/34910653/176011655-b61c6c31-75a5-4365-9d17-90800a771cb0.png)
- https://github.com/shedaniel/RoughlyEnoughItems/pull/846
![](https://pbs.twimg.com/media/FV3sVhJagAAMdWB?format=jpg&name=large)
- https://github.com/shedaniel/RoughlyEnoughItems/issues/864
![2022-06-28_02 35 00](https://user-images.githubusercontent.com/34910653/176011740-ae2e0901-968f-4e28-a479-618c8d35495f.png)
- https://github.com/shedaniel/RoughlyEnoughItems/issues/908
- https://github.com/shedaniel/RoughlyEnoughItems/issues/910
![2022-06-28_02 35 46](https://user-images.githubusercontent.com/34910653/176011890-b62d1fbf-83ca-4b47-886f-98938734738b.png)
- https://github.com/shedaniel/RoughlyEnoughItems/issues/853
- Cheating via Hotbar Keybind
- Make gamemode favorites not pinned by default
- https://github.com/shedaniel/RoughlyEnoughItems/issues/939
![2022-06-28_02 36 53](https://user-images.githubusercontent.com/34910653/176012107-55b5c772-b345-4dfb-aa6b-14f214a850c5.png)
- Fixed many more bugs (View #931)

**Comment by wrldspawn** [comment]
[Give credit where credit is due...](https://github.com/emilyploszaj/emi)

**Reply by shedaniel** [reply]
No? Tag Trees are conceptualised back in April **last year**, see the original issue
